components/engine_traits/src/metrics_flusher.rs

```diff
@@ -47,6 +48,7 @@ impl<K: KvEngine, R: RaftEngine> MetricsFlusher<K, R> {
                 while let Err(mpsc::RecvTimeoutError::Timeout) = rx.recv_timeout(interval) {
                     kv_db.flush_metrics("kv");
                     raft_db.flush_metrics("raft");
+                    flush_io_metrics();
```

Comment:
is there a better place to flush io metrics? this logic is not engine specific.

Reply:
seems it's better to flush io metrics in io limiter. I'll leave a todo here, let's keep it now.